cfgov/agreements/management/commands/util.py

```diff
@@ -1,64 +1,71 @@
 import os
 
-from agreements.models import Agreement, Issuer
+from agreements.models import Issuer
+import boto3
+
 from slugify import slugify
 
 
-def clear_tables():
-    Agreement.objects.all().delete()
+def s3_safe_key(path, prefix=''):
+    key = prefix + path.encode('utf-8')
```

Comment:
I don't think we need to encode the path at this point; AWS says keys should be delivered as unicode, and the uploads work for me without it.
Also, I would remove comma in addition to the other chars.

Reply:
Latest commit addresses this